### PR TITLE
✨ add confirm page

### PR DESCRIPTION
**Pull request by mukappalambda**

### CODE REVIEW

File: pkg/pages/confirm.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	selectedItemStyle = lipgloss.NewStyle().PaddingLeft(2).Foreground(lipgloss.Color("170"))
+	paginationStyle   = list.DefaultStyles().PaginationStyle.PaddingLeft(4)
+	helpStyle         = list.DefaultStyles().HelpStyle.PaddingLeft(4).PaddingBottom(1)
+	quitTextStyle     = lipgloss.NewStyle().Margin(1, 0, 2, 4)
+)
+
+type ConfirmPage struct {
+	list        list.Model
+	DB          *sql.DB
+	saveSession int
```
Comment (kevinliao852) [inline pkg/pages/confirm.go:27]: nit: can be *int

File: pkg/pages/confirm.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+		switch msg.Type {
+		case tea.KeyEnter:
+			i, ok := c.list.SelectedItem().(item)
+			if ok {
+				if string(i) == "Yes" {
+					c.saveSession = 1
+					log.Println("Save to config")
+				}
+				if string(i) == "No" {
+					c.saveSession = 0
```
Comment (kevinliao852) [inline pkg/pages/confirm.go:102]: nit:  could use enum instead

**Review by kevinliao852 — APPROVED** [review]

File: pkg/pages/confirm.go (hunk 1 above)
Comment (mukappalambda) [reply pkg/pages/confirm.go:27]: #16

File: pkg/pages/confirm.go (hunk 2 above)
Comment (mukappalambda) [reply pkg/pages/confirm.go:102]: #17

**Pull request merged**